repos/fdbt-site/src/pages/editPeriodDuration.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { ReactElement } from 'react';
+import TwoThirdsLayout from '../layout/Layout';
+import { ErrorInfo, NextPageContextWithSession } from '../interfaces';
+import ErrorSummary from '../components/ErrorSummary';
+import CsrfForm from '../components/CsrfForm';
+import { getCsrfToken } from '../utils';
+import ExpirySelector from 'src/components/ExpirySelector';
+import { getSessionAttribute } from 'src/utils/sessions';
+import { EDIT_PERIOD_DURATION_ERROR, MATCHING_JSON_ATTRIBUTE } from 'src/constants/attributes';
+import { ExpiryUnit } from 'src/interfaces/matchingJsonTypes';
+
+const title = 'Edit Period Duration - Create Fares Data Service';
+const description = 'Edit Period Duration page of the Create Fares Data Service';
+
+interface EditPeriodDurationProps {
+    errors?: ErrorInfo[];
+    csrfToken: string;
+    productDurationValue: string;
+    productDurationUnit: string;
+}
+
+const EditPeriodDuration = ({
+    errors = [],
+    csrfToken,
+    productDurationValue,
+    productDurationUnit,
+}: EditPeriodDurationProps): ReactElement => {
+    return (
+        <TwoThirdsLayout title={title} description={description} errors={errors}>
+            <CsrfForm action="/api/editPeriodDuration" method="post" csrfToken={csrfToken}>
+                <div className="govuk-form-group">
+                    <ErrorSummary errors={errors} />
+                    <h1 className="govuk-heading-l" id="multiple-product-page-heading">
+                        Edit Period Duration
+                    </h1>
+
+                    <>
+                        <label className="govuk-label" htmlFor="product-details-period-duration">
+                            Period duration
+                        </label>
+                        <span className="govuk-hint" id="product-period-duration-hint">
+                            For example, 3 days
+                        </span>
+                    </>
+                    <ExpirySelector
+                        defaultDuration={productDurationValue || undefined}
+                        defaultUnit={(productDurationUnit as ExpiryUnit) || undefined}
+                        quantityName={'multipleProductDurationInput'}
+                        quantityId={'product-details-period-duration-quantity'}
+                        hintId="product-period-duration-hint"
+                        unitName={'multipleProductDurationUnitsInput'}
+                        unitId={`product-details-period-duration-unit`}
+                        carnet={false}
+                        errors={errors}
+                        school={false}
```

Comment:
How do we know school and carnet are false?

Reply:
because carnet doesn't have period duration - school could have it may need to review that

Reply:
as discussed to be set to false